rest_framework/utils/model_meta.py

```diff
@@ -146,7 +152,11 @@ def _get_reverse_relationships(opts):
         )
 
     # Deal with reverse many-to-many relationships.
-    for relation in opts.get_all_related_many_to_many_objects():
+    if django.VERSION < (1, 9):
```

Comment:
> Do you want to move all compat-related things to compat.py or just the latest addition?
Just stuff that has a version switch.

> It's probably better to use django version check instead of getattr.
Agree. The version check is fine.